src/face_position_cache.cpp

```diff
@@ -21,14 +21,14 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "threading/mutex_auto_lock.h"
 
 
-UNORDERED_MAP<u16, std::vector<v3s16> > FacePositionCache::cache;
+std::unordered_map<u16, std::vector<v3s16> > FacePositionCache::cache;
```

Comment:
Either
`namespace::container<typename, namespace::container<typename>>` or
`namespace::container<typename, namespace::container<typename> >`

But you mixed both notations in this PR.